charts/argo/templates/server-deployment.yaml

```diff
@@ -63,6 +63,14 @@ spec:
             value: {{ .Values.server.baseHref | quote }}
           resources:
             {{- toYaml .Values.server.resources | nindent 12 }}
+          volumeMounts:
+          {{- if .Values.server.volumeMounts }}
+  {{- toYaml .Values.server.volumeMounts | nindent 12}}
```

Comment:
Mind indenting these to where they would normally sit? potentionaly using a `with` block like the done for `tolerations`?

Reply:
done